modules/search/github.py

```diff
@@ -36,19 +36,23 @@
 def search(self, name, q, q_formats, limit, count):
 	global PAGES,LINKS
 	engine = getattr(self, name)
+	eng = name
+	q = q_formats[f"{name}"] if f"{name}" in q_formats else q_formats['default']
 	name = engine.__init__.__name__
-	q = f"{name}_q" if f"{name}_q" in q_formats else q_formats['default_q']
 	varnames = engine.__init__.__code__.co_varnames
 	if 'limit' in varnames and 'count' in varnames:
 		attr = engine(q, limit, count)
 	elif 'limit' in varnames:
 		attr = engine(q, limit)
 	else:
 		attr = engine(q)
-
-	attr.run_crawl()
-	LINKS += attr.links
-	PAGES += attr.pages
+	if eng == 'github':
+		run = self.github(q)
+		run.run_crawl()
```

Comment:
It doesn't return anything and therefore I don't see any results.
Also you could try to figure a way to include `github` engine in the above 2 if statements and not run a separate `if`

Reply:
I have tried various permutations. It couldn't be done. Do make the changes, I will follow it next time.